downstream/assemblies/platform/assembly-using-rhsso-operator-with-automation-hub.adoc

```diff
@@ -0,0 +1,37 @@
+ifdef::context[:parent-context: {context}]
+
+[id="using-rhsso-operator-with-automation-hub"]
+
+:context: using-a-rhsso-operator
+
+= Using a Red Hat Single Sign On Operator with {HubName}
+
+The _{PlatformName}_ (AAP) _{PrivateHubNameStart}_ (PAH) uses Red Hat Single Sign-On (RH-SSO) for external authentication.
```

Comment:
```suggestion
The _{PlatformName}_ (AAP) _{PrivateHubNameStart}_ (PAH) uses Red Hat Single Sign-On (RH-SSO) for authentication.
```

Reply:
Not sure what is meant by "external" authn. I guess you could say "user authentication" but just authentication seems sufficient.